BUILD.bazel

```diff
@@ -45,6 +45,34 @@ java_binary(
     ],
 )
 
+java_binary(
```

Comment:
@miraleung (from https://github.com/googleapis/gapic-generator-java/pull/767): Nit: Can we add a summary comment above these two rules, which also indicate these are debugging tools?

Reply:
These are not rules, these are java_binaries used by one of the rules only. I clarified the purpose of the binaries in the comments

Reply:
Thanks for adding the comments. Could we copy-paste these into the Java src files as well? Up to you whether to replace these comments with a pointer to the Java source, or leave these as they are.

Could we also add a usage example? Up to you whether that goes here or in the Java source comments.

Reply:
Done